ERROR:daisy.client:Block downscale/9881 with read ROI [0:3200, 256:512, 9472:9728] (3200, 256, 256) and write ROI [0:3200, 256:512, 9472:9728] (3200, 256, 256) failed in worker 53
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
ERROR:daisy.client:Block downscale/1146 with read ROI [0:3200, 768:1024, 3840:4096] (3200, 256, 256) and write ROI [0:3200, 768:1024, 3840:4096] (3200, 256, 256) failed in worker 53
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
ERROR:daisy.client:Block downscale/9149 with read ROI [0:3200, 1024:1280, 8448:8704] (3200, 256, 256) and write ROI [0:3200, 1024:1280, 8448:8704] (3200, 256, 256) failed in worker 53
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
ERROR:daisy.client:Block downscale/701 with read ROI [0:3200, 1536:1792, 2304:2560] (3200, 256, 256) and write ROI [0:3200, 1536:1792, 2304:2560] (3200, 256, 256) failed in worker 53
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
